### DISCUSSION TITLE
Ghost functionalities.

### DISCUSSION THREAD

**Discussion by Ordinatos** [sumatrapdfreader/sumatrapdf#3036] [Q&A]
Hello everybody.
My question is probably naive but I didn't find the answer in the SumatraPDF Documentation nor in the forum.

I looked at the different commands on the page https://www.sumatrapdfreader.org/docs/Commands
and I opened the command palette (Ctrl+k) in a pdf file opened in SumatraPDF.
Among the listed features some of them seem to me not to work.
Example (among others): 
Create Polygon Annotation (CmdCreateAnnotPolygon)
If I run this command (double click in the CP) nothing happens.

There must be something I don't understand, but what?
Cap

**Comment by GitHubRulesOK** [comment]
hmm Well spotted 
those 3 should have returned NYI (Not yet implemented)
![image](https://user-images.githubusercontent.com/12754913/197075481-639c9024-af9d-46e0-875e-72364257718f.png)

@kjk could/should be others ?
